client-react/src/components/form-controls/TextFieldNoFormik.tsx

```diff
@@ -72,11 +72,7 @@ const TextFieldNoFormik: FC<ITextFieldProps & CustomTextFieldProps> = props => {
   };
 
   const getTextFieldStyles = () => {
-    if (!!styles && typeof styles === 'function') {
-      return styles;
-    } else {
-      return textFieldStyleOverrides(theme, fullpage, widthOverride, styles);
-    }
+    return styles ?? textFieldStyleOverrides(theme, fullpage, widthOverride);
```

Comment:
Consider using a `useCallback` hook since this function depends on props.

```suggestion
  const getTextFieldSytles = React.useCallback(() => {
    return styles ?? textFieldStyleOverrides(theme, fullpage, widthOverride);
  }, [styles, theme, width, widthOverride]);
```

Reply:
Done